src/groups/mwc/mwcio/mwcio_ntcchannelfactory.t.cpp

```diff
@@ -658,10 +661,9 @@ void Tester::connect(int                          line,
 
     ConnectOptions reqOptions(options, s_allocator_p);
 
-    reqOptions.setAttemptInterval(
-        bsls::TimeInterval(0, 10 * bdlt::TimeUnitRatio::k_NS_PER_MS));
-
-    reqOptions.setAttemptInterval(bsls::TimeInterval(0, 1));
```

Comment:
I'm not sure I understand why the second call of `setAttemptInterval` with such strange arguments (1 ns) is required.